tgui/packages/tgui/interfaces/MinimapController.js

```diff
@@ -0,0 +1,245 @@
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, ByondUi, Dropdown, Flex, Image, Input, Modal, NumberInput, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { capitalize } from './common/stringUtils';
+
+export const MinimapController = (params, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    title,
+    theme,
+    minimap_id,
+    markers_visible,
+    selecting_coordinates,
+    minimap_markers,
+    showNewMarkerMenu,
+    placable_marker_states,
+    placable_marker_images,
+    icon,
+    image,
+    x,
+    y,
```

Comment:
Prefer `xPos` or `yPos`. Minor, but usage of `x` as a variable name further down had me checking if it had been used as just a "do the thing" variable (e.g. a callback in an array map, `.map((x) => { ... some stuff here that uses `x` })`. That wouldn't be okay either, as better variable names are better, but it's certainly not uncommon so has me on guard.